src/modules/paymentProcessor/StreamingPaymentProcessor.sol

```diff
@@ -209,10 +200,10 @@ contract StreamingPaymentProcessor is Module, IStreamingPaymentProcessor {
     }
 
     /// @inheritdoc IStreamingPaymentProcessor
-    function removePayment(IPaymentClient client, address contributor)
-        external
-        onlyAuthorized
-    {
+    function removeAllPaymentForContributor(
```

Comment:
typo: use removeAllPaymentsForContributor instead.

Reply:
Fixed. Went ahead and renamed.